christianR15c/kisi-backend
# github action CI

#### What does this PR do?

- automate integration through github actions

#### Description of Task to be completed?

- CI added using github actions

#### How should this be manually tested?

- when you push any branch on this repo or raised a PR you should see that it will run tests first
